WICS-UCI/venushacks
# Add 2022 Sponsors to Home Page

## Summary
Update sponsors to 2022.
![image](https://user-images.githubusercontent.com/29494270/168904876-fab03f03-0f90-4032-9e72-4b777eb46a2a.png)

I moved the old sponsors to a `/past-sponsors` folder incase we want to use those logos on a sponsors page in the future.

## Test Plan
- View the site on desktop and mobile
- Verify that the logos look good
- Verify that clicking on each logo takes us to the company website

## Issue(s)
Related to #121 

## Future Followup
Create dedicated sponsors page for #121
